app/controllers/consent_forms_controller.rb

```diff
@@ -8,12 +8,19 @@ def start
   end
 
   def create
+    vaccine = @session.campaign.vaccines.first
+    first_health_question = vaccine.first_health_question
     health_answers =
-      @session.campaign.vaccines.first.health_questions.map do |hq|
-        HealthAnswer.new question: hq.question,
+      [
+        first_health_question,
+        *(vaccine.health_questions - [first_health_question])
```

Comment:
We could extract this as a `rest_of_health_questions` variable, might be a bit clearer what the intent is

Minor, feel free to ignore

Reply:
Alternatively it can be

```ruby
(vaccine.health_questions - [first_health_question]).unshift(first_health_question)
```

Not sure that's necessarily better though